### PR TITLE
Change invariant for amount in query

### PR DESCRIPTION
**Pull request by neewy**
### Description of the Change

I tested JS wallet today. When admin did not have assets with positive balance, there were no wallets (yet there were assets, just with 0 balance).  

### Benefits

I have corrected this output so that now use is returned empty assets, as in here:
```
account_assets_response {
  account_assets {
    asset_id: "coin#domain"
    account_id: "admin@test"
    balance {
      value {
      }
      precision: 2
    }
  }
}

```
### Possible Drawbacks 

None

### Usage Examples or Tests *[optional]*

get_account_assets_test

### CODE REVIEW

File: shared_model/validators/field_validator.cpp [hunk 1]
```diff
@@ -106,7 +106,7 @@ namespace shared_model {
 
     void FieldValidator::validateAmount(ReasonsGroupType &reason,
                                         const interface::Amount &amount) const {
-      if (amount.intValue() <= 0) {
+      if (amount.intValue() < 0) {
```
Comment (l4l) [inline shared_model/validators/field_validator.cpp]: That seems to be always true

File: shared_model/validators/field_validator.cpp [hunk 2]
```diff
@@ -106,7 +106,7 @@ namespace shared_model {
...
         auto message =
             (boost::format("Amount must be greater than 0, passed value: %d")
```
Comment (l4l) [inline shared_model/validators/field_validator.cpp]: Description doesn't match now

File: test/integration/acceptance/get_account_assets_test.cpp [hunk 3]
```diff
@@ -50,14 +50,20 @@ class GetAccountAssets : public AcceptanceFixture {
     return complete(baseQry().queryCounter(1).getAccountAssets(kUserId));
   }
 
-  const std::string kNewRole = "rl";
-};
+  static auto checkAccountAssets(int quantity) {
+    return
```
Comment (l4l) [inline test/integration/acceptance/get_account_assets_test.cpp]: is it formatted?

File: test/integration/acceptance/get_account_assets_test.cpp [hunk 4]
```diff
@@ -66,6 +72,8 @@ auto checkValid = [](auto &status) {
  * @then there is an AccountAssetResponse
  */
 TEST_F(GetAccountAssets, AddedAssets) {
+  auto check_single_asset = checkAccountAssets(1);
```
Comment (l4l) [inline test/integration/acceptance/get_account_assets_test.cpp]: maybe inline at 85th line?
same for the following

**Comment by neewy** [comment]
There were issues with this PR since I have changed the interface in improper way.
I've just changed the test and disabled it.

**Pull request merged**